src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -92,81 +92,80 @@
 
 (defn user-message-content
   [{:keys [content-type quoted-message content outgoing outgoing-status] :as message-data}
-   {:keys [chat-id] :as context}]
-  [:f>
-   (let [show-delivery-state? (reagent/atom false)]
-     (fn []
```

Comment:
you can remove :f but you have to keep atom here and keep render function, just move all parameters from component to render function

Reply:
Keeping atom here without `:f>` broken showing message status

Reply:
ok then we should fix that :)

Reply:
The fix I got on the top of my head was to move the atom in it's parent

Reply:
you have to keep atom here and keep render function, just move all parameters from component to render function

Reply:
https://media.tenor.com/9hd9gB_WFqIAAAAC/sniper-pubg.gif

Reply:
<img src="https://media.tenor.com/9hd9gB_WFqIAAAAC/sniper-pubg.gif"/>